src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/PresenterDtoValidatorTest.java

```diff
@@ -34,84 +34,84 @@ public void init() {
 
     @Test
     void testNoErrorReportedWhenPresenterDtoValuesAreCorrect() {
-        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), CONTEXT);
+        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), LOGGING_CONTEXT);
         assertFalse(errors.hasErrors());
     }
 
     @Test
     void testErrorReportedWhenNameFieldIsEmpty() {
         presenterDto.setFullName("  ");
-        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), CONTEXT);
+        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), LOGGING_CONTEXT);
         String qualifiedFieldName = getQualifiedFieldName(PresenterDto.FULL_NAME_FIELD);
-        String validationMessage = ValidationMessages.NOT_EMPTY_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        String validationMessage = String.format(ValidationMessages.NOT_EMPTY_ERROR_MESSAGE, qualifiedFieldName);
 
         assertError(PresenterDto.FULL_NAME_FIELD, validationMessage, errors);
     }
 
     @Test
     void testErrorReportedWhenNameFieldIsNull() {
         presenterDto.setFullName(null);
-        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), CONTEXT);
+        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), LOGGING_CONTEXT);
         String qualifiedFieldName = getQualifiedFieldName(PresenterDto.FULL_NAME_FIELD);
-        String validationMessage = ValidationMessages.NOT_NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
 
         assertError(PresenterDto.FULL_NAME_FIELD, validationMessage, errors);
     }
 
     @Test
     void testErrorReportedWhenNameFieldExceedsMaxLength() {
-        presenterDto.setFullName(StringUtils.repeat("A", 161));
-        Errors errors = presenterDtoValidator.validate(presenterDto, new Errors(), CONTEXT);
+        presenterDto.setFullName(StringUtils.repeat("A", 257));
```

Comment:
I'd also add tests to check no error reported when length is equal to max length